packages/ui/src/components/Switch/android/styles.scss

```diff
@@ -64,12 +120,24 @@ $switch-l-size: $root-l-height;
   box-sizing: border-box;
   transition: background-color 0.2s ease, left 0.3s ease, border 0.2s ease;
   background-color: white;
-  border: solid #AEB5BB;
+  border: 3px solid #AEB5BB;
```

Comment:
Probably, my fault. We should define border size here. Its size should be implemented by `switch-size` mixin as long as it will depend on switch size.

Reply:
I don't think we need it. Every default size of borders looks great with different size of this object. 
But if you wish, I will do it.